README.md

```diff
@@ -408,6 +389,7 @@ func main() {
     config.DisableTOML()
```

Comment:
What do you think of functional options 
config.Load(&config{}, config.OnlyJSON())

Reply:
Not a bad idea. Let me think about it. I like how compact that is. I want to make sure it's intuitive enough that the user will either remember it after viewing the docs and easily see what's going on if it's someone who hasn't used the lib before. Perhaps both could be used?

Reply:
Maybe chaining isn't out of the question either?

```go
config.Load(&cfg{}).OnlyJSON()
```

Reply:
If the options are functions on the package or chained methods then someone with intelliSense could see the options without needing to consult the docs too much.